server/controllers/login.js

```diff
@@ -0,0 +1,28 @@
+var model = require("../models")
+const { User } = model;
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const SECRET_KEY = "secretkey224400";
+
+class Login {
+    static async create(req, res) {
+        const { email, password } = req.body
+        let allowLogin = false
+        const users = await User.findAll({
+            where:{ email }
+        })
+
+        if (users && users.length > 0)
+            allowLogin = bcrypt.compareSync(password, users[0].password)
+        
+        if(!allowLogin)
+            return res.status(401).send({message: 'Email and Password does not match!'});
+
+        const expiresIn = 24 * 60 * 60;
+             const accessToken = jwt.sign({ id: users.id }, SECRET_KEY, {
+                 expiresIn: expiresIn
```

Comment:
Already mentioned that you don't need to repeat an object key to value when they are the same